includes/templates/template_default/templates/tpl_modules_shipping_estimator.php

```diff
@@ -48,9 +48,8 @@
     </address>
     <br class="clearBoth">
 <?php
-} else {
-    if ($_SESSION['cart']->get_content_type() !== 'virtual') {
-        $flag_show_pulldown_states = (ACCOUNT_STATE_DRAW_INITIAL_DROPDOWN === 'true');
+} elseif ($_SESSION['cart']->get_content_type() !== 'virtual') {
```

Comment:
i frequently do multiple commits; first correct the issue, and then reformat/refactor.  here you have done the reverse...  of which i thought about; and of which i can be converted to be a fan.

that said, this section of code looks to be a refactor/reformat; and not the part of the actual correction.

it would be great to just look at the actual correction and not superfluous stuff in a commit labeled as such.  

because for the rest of this commit, i have no idea where the correction now is...  nor am i now willing to now commit the time to finding it.

Reply:
Yep, me too, I just "got on a roll" and saw those couple of other refinements when I made the 'actual' change.